lib/wallet.js

```diff
@@ -315,6 +315,144 @@ class WalletClient extends Client {
     return this.get(`/wallet/${id}/coin/${hash}/${index}`);
   }
 
+  /**
+   * Get name state for the given name.
+   * {@see hsd.NameState}
+   * @param {String} id
+   * @param {String} name
+   * @returns {Promise}
+   */
+
+  getName(id, name) {
+    return this.get(`/wallet/${id}/name/${name}`);
+  }
+
+  /**
+   * Get name state for all names
+   * that the wallet is managing.
+   * {@see hsd.NameState}
+   * @param {String} id
+   * @param {Object} options
+   * @returns {Promise}
+   */
+
+  getNames(id, options) {
+    return this.get(`/wallet/${id}/name`, options);
+  }
+
+  /**
+   * Get bids, reveals and name state
+   * for the given name.
+   * {@see hsd.NameState}
+   * {@see hsd.BlindBid}
+   * {@see hsd.BidReveal}
+   * @param {String} id
+   * @param {String} name
+   * @returns {Promise}
+   */
+
+  getAuctionByName(id, name) {
+    return this.get(`/wallet/${id}/auction/${name}`);
+  }
+
+  /**
+   * Get bids, reveals and name state
+   * for all names the wallet manages.
+   * {@see hsd.NameState}
+   * {@see hsd.BlindBid}
+   * {@see hsd.BidReveal}
+   * @param {String} id
+   * @param {Object} options
+   * @returns {Promise}
+   */
+
+  getAuctions(id, options) {
```

Comment:
I do not see a corresponding HTTP endpoint for this client method.

Reply:
Good catch, the endpoint is added in https://github.com/handshake-org/hsd/pull/163/commits/6bdfd1f7a2b6412ccacfc6f00569bee79439861e